hip-service-test/Discovery/Builder/IdentifierBuilder.cs

```diff
@@ -0,0 +1,23 @@
+namespace hip_service_test.Discovery
+{
+    using HipLibrary.Patient.Models;
+
+    // ReSharper disable once ClassNeverInstantiated.Global
+    internal class IdentifierBuilder
+    {
+        private IdentifierType Type { get;  }
+
+        private string Value { get; }
+
+        public Identifier build()
+        {
+            return new Identifier(Type, Value);
+        }
+
+        // ReSharper disable once EmptyConstructor
+        public IdentifierBuilder()
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this redundant constructor.](https://app.codacy.com/gh/ProjectEKA/hip-service/pullRequest?prid=4780538)